Source/Core/DiscIO/RiivolutionParser.cpp

```diff
@@ -273,13 +275,13 @@ std::vector<Patch> Disc::GeneratePatches(const std::string& game_id) const
         while (!sv.empty())
         {
           bool replaced = false;
-          for (const auto& r : replacements)
+          for (const auto& [first, second] : replacements)
```

Comment:
I think this could use better variable names, `substring` and `replacement` maybe?
There's a similar situation below.